sdk/core/azure-core/inc/azure/core/internal/keyvault_exception.hpp

```diff
@@ -31,4 +31,4 @@ namespace Azure { namespace Security { namespace KeyVault { namespace _internal
     static Azure::Core::RequestFailedException CreateException(
```

Comment:
@JeffreyRichter should we just make this a public ctor on RequestFailedException or leave it as an `_internal` static factory method for now? If we want to leave it as static, any suggestions on a name for the containing class (because we certainly don't want it to be `KeyVaultException`).

Reply:
Since we already have public ctors, I'd just add another overload and call it good.